### DISCUSSION TITLE
Repeatedly Marking Something as "Watched" on Plex

### DISCUSSION THREAD

**Discussion by PaxVobiscuit** [Taxel/PlexTraktSync#638] [Q&A] [answered]
Whenever I run the tool, I have two movies that it repeatedly marks as watched on Plex.   it is true, I have watched both movies, but why is it marking them that way every time I run the tool?

**Comment by PaxVobiscuit** [comment, marked as answer]
Once again answering my own question after the fact - I deleted the local cache DB and reran and now it is working as expected.

**Reply by thedinz** [reply]
Thank you, the same issue for me and this fixed it, although I hope its not something that needs to be done often.
